core/server/api/app.js

```diff
@@ -11,7 +11,7 @@ var debug = require('debug')('ghost:api'),
     // API specific
     auth = require('../auth'),
     cors = require('../middleware/api/cors'),   // routes only?!
-    spamPrevention = require('../middleware/api/spam-prevention'), // routes only
```

Comment:
I can't find any occurrences for the old middleware?Is it still used?
If not, it would be great to remove it and keep the unit tests for it, but just used with the new logic.